.gitignore

```diff
@@ -4,6 +4,10 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+# ignore vscode stuff
```

Comment:
This is unrelated to URL change. Please, put it to separate commit.

Reply:
Moved to another commit